front-end/src/components/TranscriptUploadModal.jsx

```diff
@@ -40,12 +40,11 @@ class TranscriptUploadModal extends React.Component {
   }
 
   async handleUpload(file) {
-    const result = await TranscriptAPI.post(file);
-    if(result.status === 500){
-      window.alert("Error in uploading transcript. Please try again.")
-    }
-    else {
+    try {
+      await TranscriptAPI.post(file);
       this.props.toggleModal();
+    } catch (e) {
+      window.alert("Error in uploading transcript. Please try again.");
```

Comment:
Maybe display the error message. Something like

```suggestion
      window.alert(`Error in uploading transcript. ${e.message}`);
```
LMK if you think it's a good idea

Reply:
I did add in the error message and it's showing result returned with error 500. Not sure exactly how useful that is.

Reply:
Fair enough. I'm fine with either version.